### PR TITLE
use puppeteer for baking

### PR DESCRIPTION
**Pull request by yahengsu**
This change attempts to solve https://github.com/webaverse/xrpackage/issues/55, using puppeteer to spawn a headless browser for the baking process. For debugging purposes, setting the `headless: true` option to `headless: false` will spawn a viewable browser window.

Testing `npm start bake ...` against `xrpk bake ...` and the screenshots/volumization/aabb seem to produce the same results.

### CODE REVIEW

**Comment by yahengsu** [comment]
We noticed that open() was being used in other files (`inspect.js`, `run.js`, `volume.js`, `util.js`), should we leave these as-is or consider transitioning these files to use puppeteer as well?

**Comment by avaer** [comment]
Some of these make sense to be moved and some do not. For example, run probably open the user's browser to actually test the package; loading it in a headless browser wouldn't be particularly useful.

File: commands/bake.js [hunk 1]
```diff
@@ -60,9 +59,15 @@ const _bakeApp = async output => {
   const server = http.createServer(app);
   const connections = [];
   server.on('connection', c => connections.push(c));
-  server.listen(port, () => {
-    open(`https://xrpackage.org/bake.html?srcWbn%3Dhttp://localhost:${port}/a.wbn%26dstGif%3Dhttp://localhost:${port}/screenshot.gif%26dstVolume%3Dhttp://localhost:${port}/volume.glb%26dstAabb%3Dhttp://localhost:${port}/aabb.json`);
-  });
+  server.listen(port);
```
Comment (avaer) [inline commands/bake.js:62]: I think if you don't wait for the listen, there is a race condition.

**Comment by avaer** [comment]
I would move both screenshot and volume commands to the puppeteer (since they are just sub-parts of the bake).

**Comment by avaer** [comment]
Published as `xrpk@0.0.74`.

**Pull request merged**